### DISCUSSION TITLE
unsupported or empty accounts for namespace

### DISCUSSION THREAD

**Discussion by damianlluch** [WalletConnect/walletconnect-monorepo#4293] [Developer Support]
Hi, I'm getting this error with WalletConnectV2, (using web3-react) after approving a connection from Trust Wallet.

![IMG_C9F078BD97FA-1](https://github.com/WalletConnect/walletconnect-monorepo/assets/1463804/68553a9f-d72b-4b46-ae2d-dbc21388b301)

Any suggestions?

Thanks

**Comment by glitch-txs** [comment]
Hi, could you share a minimal reproducible example and steps to reproduce the issue? Thanks!

**Reply by Lil23d** [reply]
Have the same issue too 
![IMG_1942](https://github.com/WalletConnect/walletconnect-monorepo/assets/164969302/1f1b58cc-a4ac-4bfb-a923-60c95fdb50d0)
